src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -193,6 +194,7 @@
     <Compile Include="ReadOnlyMemory\Equality.cs" />
     <Compile Include="ReadOnlyMemory\GetHashCode.cs" />
     <Compile Include="ReadOnlyMemory\ImplicitConversion.cs" />
+    <Compile Include="ReadOnlyMemory\Pin.cs" />
```

Comment:
corefx itself uses Retain in a bunch of places.  Can those be switched over to Pin in this PR as well?

Reply:
Yes, in progress.

Reply:
Done